assignment01-voronoi/tiles.h

```diff
@@ -0,0 +1,99 @@
+/*
+   Tile Definitions for wave fuction collapse algorithm
+*/
+
+#ifndef _TILES_H
+#define _TILES_H
+
+#define BLK "\e[0;30m"
+#define RED "\e[0;31m"
+#define GRN "\e[0;32m"
+#define YEL "\e[0;33m"
+#define BLU "\e[0;34m"
+#define MAG "\e[0;35m"
+#define CYN "\e[0;36m"
+#define WHT "\e[0;37m"
```

Comment:
don't define these here tiles have nothing to do with what is printed out

Reply:
had them there initially to test, but then left them as to not clutter worldgen.c, should i make a new header with only them in it?